src/test/java/org/broadinstitute/consent/http/resources/UserResourceTest.java

```diff
@@ -518,7 +519,7 @@ public void testGetUsersByInstitutionSuccess() {
   public void testUpdateSelf() {
     User user = createUserWithRole();
     UserUpdateFields userUpdateFields = new UserUpdateFields();
-    Gson gson = new Gson();
+    Gson gson = GsonUtil.gsonBuilderWithAdapters().create();
```

Comment:
Not a requirement for this PR. Having so many instantiations would generally suggest we create a private final `gson` at the top of this test class.